src/lib/components/prettybytes/PrettyBytes.component.tsx

```diff
@@ -2,9 +2,14 @@ import prettyBytes from 'pretty-bytes';
 type Props = {
   bytes?: number | null;
   decimals?: number | null;
+  activeBinaryPrefix?: boolean;
```

Comment:
I would prefer using a more developer friendly name, maybe a format prop with 2 possible values (iB - default and B) or base prop with 2 possible values (1000, 1024 - default). I feel like saying binary here expose too much of the choices of the underlying library.